src/common/containers/build-details.js

```diff
@@ -81,7 +81,7 @@ class BuildDetails extends Component {
         { isFetching &&
           <div className={styles.spinner}><Spinner /></div>
         }
-        { snap &&
+        { snap && snap.store_name &&
```

Comment:
There still may be a case when user sees this page before snap has a name (before name is registered), so we for sure shouldn't show the box in such case.

We still have separate card to properly hide this box until snap is successfully published.